libcontainer/init_linux.go

```diff
@@ -522,7 +522,8 @@ func isWaitable(pid int) (bool, error) {
 
 // signalAllProcesses freezes then iterates over all the processes inside the
 // manager's cgroups sending the signal s to them.
-// If s is SIGKILL then it will wait for each process to exit.
+// If s is SIGKILL and subreaper is not enabled then it will wait for each
+// process to exit.
```

Comment:
nit (and not new); try to avoid future tense in tech-docs. This could be

```suggestion
// If s is SIGKILL and subreaper is not enabled then it waits for each
// process to exit.
```